src/main/java/de/tum/in/www1/artemis/service/connectors/jenkins/dto/TestCaseDTO.java

```diff
@@ -33,4 +36,45 @@ public TestCaseDTO(String name, String classname, double time) {
     public boolean isSuccessful() {
         return CollectionUtils.isEmpty(errors) && CollectionUtils.isEmpty(failures);
     }
+
+    @Override
+    public String getName() {
+        return name;
+    }
+
+    @Override
+    public List<String> getMessage() {
+        return extractMessageFromTestCase().map(List::of).orElse(List.of());
```

Comment:
same effect but increased readablity.
```suggestion
        return extractMessageFromTestCase().map(Collections::singletonList).orElse(Collections.emptyList());
```